xml-occurrence-parsers/src/main/java/org/gbif/xml/occurrence/parser/parsing/extendedrecord/DataFileWriterProxy.java

```diff
@@ -0,0 +1,26 @@
+package org.gbif.xml.occurrence.parser.parsing.extendedrecord;
+
+import org.gbif.pipelines.io.avro.ExtendedRecord;
+
+import java.io.IOException;
+
+import org.apache.avro.file.DataFileWriter;
+
+/**
+ * Proxy class for avro DataFileWriter, created to avoid an issue during file writing
+ */
+public class DataFileWriterProxy {
```

Comment:
better renamed this class to SyncDataFileWriter??

Reply:
Fixed